src/psyclone/tests/nemo/test_files/do_while.f90

```diff
@@ -29,11 +29,11 @@
 ! OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 ! OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ! -----------------------------------------------------------------------------
```

Comment:
If we get rid of the test in `nemo_psy_test`, can we get rid of this file too?

Reply:
Done,